pavelib/utils/test/suites/pytest_suite.py

```diff
@@ -161,16 +161,19 @@ def cmd(self):
                 xdist_remote_processes = self.processes
             for ip in self.xdist_ip_addresses.split(','):
                 # The django settings runtime command does not propagate to xdist remote workers
-                custom_exports = 'export MAILCHIMP_API_KEY={MAILCHIMP_API_KEY}; export TEST_DB_HOST={TEST_DB_HOST};' \
-                                 ' export TEST_DB_USER={TEST_DB_USER}; export TEST_DB_PASSWORD={TEST_DB_PASSWORD};' \
-                                 ' export TEST_DB_NAME={TEST_DB_NAME};'\
-                    .format(
-                        MAILCHIMP_API_KEY=os.environ.get('MAILCHIMP_API_KEY'),
-                        TEST_DB_HOST=os.environ.get('TEST_DB_HOST'),
-                        TEST_DB_USER=os.environ.get('TEST_DB_USER'),
-                        TEST_DB_PASSWORD=os.environ.get('TEST_DB_PASSWORD'),
-                        TEST_DB_NAME=os.environ.get('TEST_DB_NAME'),
-                    )
+                custom_exports = """
+                 export MAILCHIMP_API_KEY={MAILCHIMP_API_KEY};
+                 export MANDRILL_API_KEY={MANDRILL_API_KEY}; export TEST_DB_HOST={TEST_DB_HOST};
+                 export TEST_DB_USER={TEST_DB_USER}; export TEST_DB_PASSWORD={TEST_DB_PASSWORD};
+                 export TEST_DB_NAME={TEST_DB_NAME};
```

Comment:
let's put every export statement in a different line. There are two export statements on line 166 and 167. It'll improve readability. And I think it would be much cleaner if we use tabs at the start, and spaces at the end of the line (instead of appending a single space at the start). I don't think we even need spaces, since we already are separating export statements with a ';'. It disturbs the 4-space indentation rule that we follow if we put a single statement at the start of every line. Hope the tabs used for indentation won't cause any problem

Reply:
custom_exports = """
                    export MAILCHIMP_API_KEY={MAILCHIMP_API_KEY};
                    export MANDRILL_API_KEY={MANDRILL_API_KEY};
                    export TEST_DB_HOST={TEST_DB_HOST};
                    export TEST_DB_USER={TEST_DB_USER};
                    export TEST_DB_PASSWORD={TEST_DB_PASSWORD};
                    export TEST_DB_NAME={TEST_DB_NAME};
                """

Reply:
^I'm suggested something like this.

Reply:
@harris-irfan Actually inside string literal if you will give a tab it will be tab inside string, when you give line break it appends `\n` in the string. But lemme try I hope `\t` won't cause any problem. As this string is finally appended in a command which executes in shell. Just doing this as it's improving readability. Ideally we should stick with single quotes, as python provides three ways to define a string and recommend to stick with one across the project.

Reply:
Thank you for going the extra mile and incorporating the suggestion. 👍  Hope you have checked that inserting \t is not causing any problems. Approving.